src/components/ItemSet/index.js

```diff
@@ -18,22 +19,32 @@ const ITEM_TYPE_COMPONENTS = {
   exoticCatalysts: MasterworkCatalyst
 };
 
-function ItemSet({
-  className,
-  inventory,
-  itemDefs,
-  setPopper,
-  setModal,
-  set,
-  objectiveInstances,
-  objectiveDefs
-}) {
-  const { name, noUi, description, sections, image } = set;
+class ItemSet extends Component{
+  constructor(props) {
+    super(props);
+    this.accordion = this.accordion.bind(this);
+  }
+  
+  accordion(){
```

Comment:
Instead of manually binding, you can instead use an arrow function for the `accordion` class function. [See this](https://medium.com/quick-code/react-quick-tip-use-class-properties-and-arrow-functions-to-avoid-binding-this-to-methods-29628aca2e25). This would remove the need for the constructor all together!


https://github.com/jasonslyvia/react-lazyload#forcecheck